examples/evm-multichain-dapp/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "evm-template",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
```suggestion
    "start": "next start"
```

Reply:
- Made suggested changes.
- Merged [vercel:canary](https://github.com/vercel/next.js) to [saurabhburade:example/evm-dapp](https://github.com/saurabhburade/next.js/tree/example/evm-dapp) for latest implementation of pnpm-lock.yaml file.